glpi/files/opt/startup.sh

```diff
@@ -32,9 +32,29 @@ done
 # Set ACL for www-data user on marketplace directory
 setfacl -m user:www-data:rwx,group:www-data:rwx "/var/www/glpi/marketplace"
 
+# forward logs files to /proc/1/fd/1
+# see https://stackoverflow.com/a/63713129
+logs=(
+    "event.log"
+    "cron.log"
+    "mail.log"
+    "php-errors.log"
+    "sql-errors.log"
+    "mail-errors.log"
+    "access-error.log"
+)
+for log in "${logs[@]}"
+do
+    if [ ! -f /var/log/glpi/$log ];
+    then
+        touch /var/log/glpi/$log
+        setfacl -m user:www-data:rwx,group:www-data:rwx /var/log/glpi/$log
+    fi
+done
+tail -f /var/log/glpi/*.log > /proc/1/fd/1 &
```

Comment:
1. `tail -F` should be used to handle files thay may be removed.
2. Error logs should be sent to the `stderr` (`/proc/1/fd/1` ?).
3. Other logs should be tailed in the `stdout`.